RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -918,6 +918,10 @@ boolean L13_towerNSTower()
 		{
 			sources = 6;
 		}
+		else if(autoEquip($item[Fourth of May Cosplay Saber]))
```

Comment:
nit: you can combine this with the foolscap-equipping condition, since that's exactly the same case

Reply:
I didn't do that because the rest of the code is not written like that, but I can if you all want me to.

Reply:
how is it the same case? foolscap is a hat and the saber is a main hand item.

actually i think `else if` is often used incorrectly in this function. it looks like it started for only mutually exclusive options. but then was expanded to any options. should probably instead use
//hats from best to worst
foreach it in $items[item1, item2, item3]
if then equip, add sources, and break;

//weapons from best to worst
foreach it in $items[item1, item2, item3]
if then equip, add sources, and break;

//familiars from best to worst
foreach fam in $familiars[fam1, fam2, fam3]
if then use
add sources
break;

etc.